README.md

```diff
@@ -98,6 +98,12 @@ return client.resolveAddress(address)
   });
 
 ```
+## Release Note
```

Comment:
Change "Release Note" to "Release Notes" (assuming future releases will also have release note information.)

Also, do we want to add a new sub-section to this indicating the release version? This way, it will be easy to add release notes for future versions.
For example:

# # # Version 21.2.1

Reply:
In individual releases, we have release note section, there we can mention specific release notes.
We need to mention this specific change in readme file itself.